sharedbuffers/mapped_struct.py

```diff
@@ -947,6 +993,24 @@ def _mix_hash(code1, code2):
 @cython.locals(hval=cython.ulonglong, trunc_key=cython.longlong,
     truncated=cython.bint, flt=cython.double, mant=cython.double)
 def _stable_hash(key):
+    """
+    Compute a hash for object ``key`` in a way that is portable across processes and implementations.
```

Comment:
I thought about making it public, because I need to in order to have a clear definition of what is hashable. But I didn't want to make functional changes in a documentation PR.